charts/grafana-agent/chart/Chart.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: grafana-agent
+description: A Helm chart for Grafana agent custom resources
+type: application
+version: 0.1.0
```

Skal du ikke pege på et helm chart? 

f.eks.
```suggestion
version: 0.1.0

dependencies:
- name: grafana-agent-operator
  version: "0.1.5"
  repository: "https://grafana.github.io/helm-charts"
```